tools/generate_update_image.sh

```diff
@@ -67,11 +67,31 @@ function linkInStageing(){
     pushd ${STAGEING_DIR} 1> /dev/null
 
     mkdir assets
-    pushd assets 1> /dev/null
+    pushd assets 1> /dev/null # assets/
+
     ln -s ../../sysroot/sys/current/assets/fonts
     ln -s ../../sysroot/sys/current/assets/images
     ln -s ../../sysroot/sys/current/assets/lang
-    popd 1> /dev/null
+
+    # Add only those mp3 files that are required for update to 1.7.0
+    mkdir audio
+    pushd audio 1> /dev/null
+    ln -s ../../../sysroot/sys/current/assets/audio/alarm
+    ln -s ../../../sysroot/sys/current/assets/audio/chimes
+    ln -s ../../../sysroot/sys/current/assets/audio/evening_reminder
+    ln -s ../../../sysroot/sys/current/assets/audio/meditation
+    ln -s ../../../sysroot/sys/current/assets/audio/prewakeup
+    mkdir bell
+    pushd bell 1> /dev/null
+    mkdir bg_sounds
+    pushd bg_sounds 1> /dev/null
+    ln -s ../../../../../sysroot/sys/current/assets/audio/bg_sounds/Ancient_Greek.mp3
+    ln -s ../../../../../sysroot/sys/current/assets/audio/bg_sounds/Woodland_Ambiance.mp3
+    popd 1> /dev/null # bg_sounds
+    popd 1> /dev/null # bell
+    popd 1> /dev/null # audio
+
+    popd 1> /dev/null # assets
```

Comment:
please describe those changes in some document at it will affect UDM

Reply:
This is done only in Harmony release branch and should not be done when UDM is implemented. After UDM is implemented bell directory should be deleted and mp3 files from new directory should be indexed after the update.

Reply:
so please write it down - what has to be done during udm version ;)

Reply:
Sure, my point was that UDM is unrelated to this change so this is not the correct place to discuss this. ;)